partner-groups/README.md

```diff
@@ -0,0 +1,23 @@
+#Partner Groups
+
```

Comment:
We probably need to talk a little bit about what partner groups are:
e.g.

- Many cloud native focused groups cover areas that may more often or not consider security, we consult for these groups, etc. create security related content.
- Groups that cover other aspects of security in detail - have groups come in and share about ... for some groups, we have representation of the partner groups in our community that give us updates during our weekly calls and inform us via issues, etc.

Maybe.. we could also add names of members that want to be a representative contact point for the various groups as well :) or we can keep that for another PR too

Reply:
The new updates should suffice to resolve this conversation.